test/stress/generate-report/generate_report.py

```
@@ -133,6 +134,88 @@ def metrics_timeseries(conn, metrics_path, metrics, labels=None, group_by=None,
+
+def parse_nodestat(path, run_date):
+    """Parses a profiler-nodestat log into per-interval CPU/load samples."""
+    text = Path(path).read_text()
+    snaps = []
+    for m in re.finditer(
+            r"=== nodestat ([\d:]+) ===\ncpu +([\d ]+)\n([\d.]+) ([\d.]+) [\d.]+ \S+",
+            text):
+        jiffies = [int(x) for x in m.group(2).split()]
+        snaps.append((_profiler_ts(m.group(1), run_date), jiffies, float(m.group(3))))
+    intervals = []
+    for (t0, j0, _), (t1, j1, load1) in zip(snaps, snaps[1:]):
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)